democracy/views/section_comment.py

```diff
@@ -161,6 +162,12 @@ class SectionCommentSerializer(BaseCommentSerializer):
     Serializer for comment added to section.
     """
 
+    section = PrimaryKeyRelatedField(
+        queryset=Section.objects.all().prefetch_related(
+            "translations",
+            "hearing__translations",
+        )
+    )
```

Comment:
This concerns the Browsable API comment POST form, which generates a select for all sections.